indico/modules/rb_new/client/js/containers/BlockingList.js

```diff
@@ -0,0 +1,34 @@
+/* This file is part of Indico.
```

Comment:
In a past meeting, we agreed we'd put these in the same file as the component. Since this is new code, I'd suggest that we do that here.